nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const app = express();
+
+app.use(express.urlencoded({
+    extended: true
+}))
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
```

Comment:
```suggestion
function getFalsyCheck(parameters, numbers) {
  const falsyInParam = !parameters.some((element) => !element);
  const falsyInNum = !numbers.some((element) => !element);
  return parameters.length > 1 && falsyInParam && falsyInNum;
}

```

Reply:
So define a function with parameters outside of requests.

Reply:
Thanks a lot! I totally forgot parameters 😄